infra/providers/libvirt/config.go

```diff
@@ -0,0 +1,13 @@
+package libvirt
+
+// Config specifies libvirt specific parameters
+type Config struct {
+	// SSHKeyPath specifies the location of the SSH private key for remote access
+	SSHKeyPath string `json:"-" yaml:"ssh_key_path" validate:"required"`
```

Comment:
Why is this field ignored during json (un-)marshaling but not during yaml?

Reply:
I'm not quite sure. I copied over the field from the gce config. It looks like the gce and azure config both have SSHKeyPath ignored, but the aws config did not ignore it.
My guess is that this was meant as a security measure by not logging the location of the private SSH key?

Reply:
Yeah, I was mostly wondering about why it is dumped during yaml marshaling :) Seems weird that if you do `json.Marshal` on this struct then this field will be dumped, but if you do `yaml.Marshal` it won't.

Reply:
Oh I see. Yeah there are quite a few inconsistencies scattered throughout the code. Hopefully @a-palchikov can comment on this once hes settled in and had a chance to review.

Reply:
Yes, JSON serialization is used for relaying the relevant attributes to `terraform apply` which the key path is not.